### DISCUSSION TITLE
Plugin for playlists

### DISCUSSION THREAD

**Discussion by nigredo13** [quodlibet/quodlibet#3831] [Q&A] [answered]
Hi,
I want to write a plugin for automatically export changed playlists.
But i can't figure out how to get a signal/event when a playlist change, i.e. tracks added/removed.
Is it possible at all ? Any clue ?

**Comment by declension** [comment, marked as answer]
Hi.
This is not officially supported, but you could listen to `changed` signals from a `PlaylistLibrary`, though note they change a lot more than you might expect (watch the debug logs), and also see #3622. Really this should probably be an event plugin method (`plugin_on_playlist_changed` perhaps) I guess but it's never come up.
